rest_framework/filters.py

```diff
@@ -152,7 +163,7 @@ def remove_invalid_fields(self, queryset, fields, view):
                 field.source or field_name
                 for field_name, field in serializer_class().fields.items()
                 if not getattr(field, 'write_only', False)
-            ]
+                ]
```

Comment:
This doesn't need to be indented another level.

This is what is causing Flake8 to fail by the way.